src/javascript/node-package.ts

```diff
@@ -1274,20 +1301,14 @@ export class NodePackage extends Component {
         let desiredVersion = currentDefinition;
 
         if (currentDefinition === "*") {
-          try {
-            const modulePath = require.resolve(`${name}/package.json`, {
-              paths: [outdir],
-            });
-            const module = JSON.parse(readFileSync(modulePath, "utf-8"));
-            desiredVersion = `^${module.version}`;
-          } catch (e) {}
-
-          if (!desiredVersion) {
+          const resolvedVersion = this.tryResolveDependencyVersion(name);
```

Comment:
Let's use the one from utils here. Given that we already know projen doesn't know the version...
```suggestion
          const resolvedVersion = tryResolveDependencyVersion(name);
```

Reply:
Makes sense :+1: